go.mod

```diff
@@ -98,14 +99,14 @@ require (
 	github.com/zclconf/go-cty-yaml v1.0.2
 	go.etcd.io/bbolt v1.3.6
 	go.opencensus.io v0.23.0
-	go.uber.org/goleak v1.1.10
-	golang.org/x/crypto v0.0.0-20210513164829-c07d793c2f9a
-	golang.org/x/oauth2 v0.0.0-20210402161424-2e8d93401602
-	golang.org/x/sys v0.0.0-20210630005230-0f9fa26af87c
-	google.golang.org/api v0.44.0
-	google.golang.org/genproto v0.0.0-20210602131652-f16073e35f0c
-	google.golang.org/grpc v1.39.1
-	google.golang.org/protobuf v1.26.0
+	go.uber.org/goleak v1.1.12
```

Comment:
This is kicking up some linter issues with the `go.sum`. I think the only reason `go.mod` is being updated here at all is because we pulled in `github.com/hashicorp/nomad/nomad/structs`? If we don't do that then we don't need to update this either.